src/vm.rs

```diff
@@ -486,6 +486,51 @@ impl Vm {
         }
     }
 
+    /// Runs an exported wasm function in a (named or active) [module instance](crate::Instance).
```

Comment:
Refine the comments: `Runs an exported wasm function in a (named or active) [module instance](crate::Instance) with a timeout setting.`

Thanks!